src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,103 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //WebElements
+    @FindBy(css = "img.avatar")
+    WebElement userAvatarIcon;
+    @FindBy(css = "i[class='fa fa-plus-circle create']")
+    WebElement plusIcon;
+    @FindBy(css = "li[data-testid='playlist-context-menu-create-simple']")
+    WebElement newPlaylistButton;
+    @FindBy(css = "input[name='name']")
+    WebElement playlistInputField;
+    @FindBy(css = "[class='del btn-delete-playlist']")
+    WebElement deleteButton;
+    @FindBy(css = "div.success.show")
+    WebElement successMsg;
+    @FindBy(xpath = "//li[contains(text(),'Edit')]")
+    WebElement editPlaylistButton;
+
+    public WebElement getUserAvatar() {
+        return userAvatarIcon;
+    }
+
+    public boolean isDisplayed() {
+        return true;
+    }
+
+    //Create a playlist
+    public void createPlaylist(String playlistName) {
+        clickPlusIcon();
+        clickNewPlaylistButton();
+        createAPlaylistName(playlistName);
+    }
+
+    public void clickPlusIcon() {
+        plusIcon.click();
+    }
+
+    public void clickNewPlaylistButton() {
+        newPlaylistButton.click();
+    }
+
+    public void createAPlaylistName(String playlistName) {
+        playlistInputField.sendKeys(playlistName);
+        playlistInputField.submit();
+    }
+
+//    //Navigate and select a playlist
+//    public void selectDesiredPlaylist(String playlistName) {
+//        click(By.xpath("//section[@id='playlists']//li[@class='playlist playlist']//a[contains(text(),'" + playlistName + "')]"));
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.